agents/check_mk_agent.aix

```diff
@@ -124,7 +124,9 @@ function run_cached {
         if (( $MTIME < $MAXAGE )) ; then
             USE_CACHEFILE=1
         fi
-        CACHE_INFO="cached($MTIME,$MAXAGE)"
+        # get file time for cache
+        FTIME=$(perl -le 'print((stat shift)[9])' $CACHEFILE)
```

Comment:
It's not a good idea to add a dependency to perl on this important part of code. If we do that, we would need to ensure, that perl is install by default on all AIX systems that are still in use - and know since which version AIX comes with perl by default.